language/diem-framework/src/lib.rs

```diff
@@ -219,14 +210,14 @@ pub fn build_stdlib_doc(with_diagram: bool) {
                 .to_string_lossy()
                 .to_string(),
         ),
-        stdlib_files().as_slice(),
-        "",
+        diem_stdlib_files_no_dependencies().as_slice(),
+        vec![move_stdlib::move_stdlib_modules_full_path()],
         with_diagram,
     )
 }
 
 pub fn build_transaction_script_doc(script_files: &[String], with_diagram: bool) {
-    build_doc(
+    move_stdlib::build_doc(
         TRANSACTION_SCRIPTS_DOC_DIR,
         STD_LIB_DOC_DIR,
```

Comment:
It is possible that those paths do not work across crates. I remember there was a way to make such paths resolve relative to the crate root, but that won't work anymore I guess.